jupyterhub/handlers/base.py

```diff
@@ -692,7 +685,10 @@ def render_template(self, name, **ns):
 
     @property
     def template_namespace(self):
-        user = self.get_current_user()
+        try:
```

Comment:
I had to add this because the call to render_template from write_error tries to use the DB 🙄 .